README.md

```diff
@@ -1,317 +1,432 @@
-	<h1>Tambo AI</h1>
-	<p>
-		<b>A React package for building AI-powered applications with generative UI, where users interact through natural language.</b>
-	</p>
-	<br>
+  <img src="assets/octo-white-background-rounded.png" width="150">
+  <h1>Tambo AI</h1>
+  <h3>AI that renders your React components</h3>
+  <p>Register your components with schemas. AI decides which ones to render and what props to pass based on natural language. The AI agent orchestrates your UI—dynamically rendering components or updating persistent ones within your layout.</p>
 </div>
 
 <p align="center">
-  <a href="https://www.npmjs.com/package/@tambo-ai/react"><img src="https://img.shields.io/npm/v/@tambo-ai/react.svg" alt="npm version"></a>
-  <a href="https://github.com/tambo-ai/tambo/blob/main/LICENSE"><img src="https://img.shields.io/github/license/tambo-ai/tambo.svg" alt="license"></a>
-  <a href="https://github.com/tambo-ai/tambo/commits/main"><img src="https://img.shields.io/github/last-commit/tambo-ai/tambo.svg" alt="GitHub last commit"></a>
+  <a href="https://www.npmjs.com/package/@tambo-ai/react"><img src="https://img.shields.io/npm/v/%40tambo-ai%2Freact?logo=npm" alt="npm version" /></a>
+  <a href="https://github.com/tambo-ai/tambo/blob/main/LICENSE"><img src="https://img.shields.io/github/license/tambo-ai/tambo" alt="License" /></a>
+  <a href="https://github.com/tambo-ai/tambo/commits/main"><img src="https://img.shields.io/github/last-commit/tambo-ai/tambo" alt="Last Commit" /></a>
   <a href="https://discord.gg/dJNvPEHth6"><img src="https://img.shields.io/discord/1251581895414911016?color=7289da&label=discord" alt="Discord"></a>
-  <a href="https://github.com/tambo-ai/tambo/stargazers"><img src="https://img.shields.io/github/stars/tambo-ai/tambo.svg?style=social" alt="GitHub stars"></a>
+  <a href="https://github.com/tambo-ai/tambo"><img src="https://img.shields.io/github/stars/tambo-ai/tambo" alt="GitHub stars" /></a>
 </p>
 
-> [!IMPORTANT]
-> 🎉 We just open-sourced the hosted backend — check out [tambo-ai/tambo-cloud](https://github.com/tambo-ai/tambo-cloud)!
+<p align="center">
+  <a href="https://docs.tambo.co">Documentation</a> •
+  <a href="https://discord.gg/dJNvPEHth6">Discord</a>
+</p>
+
+---
+
+## What is Tambo?
+
+Tambo is a generative UI framework for React. Unlike traditional React apps where you define fixed routes and layouts, generative UI means the AI dynamically decides which components to render and what props to pass based on natural language conversations.
+
+Register your components with schemas, and an AI agent renders and controls them through dialogue. Users describe what they want, your agent orchestrates your UI to make it happen.
+
+Built on **Model Context Protocol (MCP)** - a standardized protocol that lets AI models connect to external systems (databases, APIs, files) the same way. This means plug-and-play integrations with Linear, Slack, and any MCP server without building custom adapters.
```

Comment:
slight wording feedback - "Built on MCP" makes it sound like we are somehow an MCP server or something. I like other places where you've said we're MCP-native or something like that.